packages/library-base/build.gradle.kts

```diff
@@ -171,6 +171,9 @@ android {
     }
 
     buildTypes {
+        getByName("debug") {
+            consumerProguardFiles("proguard-rules-consumer-common.pro")
```

Comment:
We apply the same rule for `debug` and `release`?

Reply:
Because the proguard consumer file also needs to be packed into the `debug` artifact if we for some reason should choose to minify it.